lib/components/narrative/narrative-itineraries.js

```diff
@@ -148,6 +150,33 @@ class NarrativeItineraries extends Component {
 
     if (!activeSearch) return null
 
+    // Merge duplicate itineraries together and save multiple departure times
+    let nextIndex = 0
+    const mergedItineraries = itineraries.reduce((prev, cur) => {
+      const updated = clone(prev)
+      cur.index = nextIndex++
+
+      const dupe = updated.findIndex((itin) => itinerariesAreEqual(itin, cur))
+      // If no duplicate, push full itinerary to output
+      if (dupe === -1) {
+        updated.push(cur)
+      } else {
+        // Add only new start time to existing itinerary
+        if (!updated[dupe].extraStartTimes) {
```

Comment:
Extract var for `updated[dupe]`.

Reply:
Extracted in 7969cc8c08590f6daf18c60716df01360bcd547b

Reply:
Always scared of javascript's auto-pointers... Works perfectly until suddenly you're writing to a copy